docs/HISTORY.txt

```diff
@@ -5,7 +5,10 @@ Changelog
 2.3.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Update sliderpane for accessiblity
+  - Show always empty image alt and title attributes on a pane
+  - Remove unused descriptionfield on a pane
+  [elioschmutz]
```

Comment:
ok, but it does not do what you want:
<img width="1056" alt="bildschirmfoto 2015-10-30 um 09 58 34" src="https://cloud.githubusercontent.com/assets/7469/10841814/d6d159b0-7eec-11e5-8509-d8a3a6a299cc.png">